.github/workflows/bump-version.yml

```diff
@@ -1,8 +1,8 @@
 name: Bump Version Number
 on:
-  push:
-    branches:
-      - master
+  pull_request:
+    branches: [ master ]
+    types: [ opened ]
```

Comment:
Does that mean you have to pull your branch after opening the PR to get that change and continue working on it if needed?

Reply:
Yes, just git pull && git push. If that's annoying we can keep it as is and close this.

Reply:
I think it's better than the old system, especially if that means we can protect master and make CI passing mandatory again.

I'm not completely sold on the whole automatic version update thing in general, but this will probably make it better